400-claims.md

```diff
@@ -128,64 +128,105 @@ The parameter data stored in a claim data is _the resolved key/value pairs_ that
   - Valid user-supplied values are presented
   - Default values are supplied for all parameters where `default` is provided and no user-supplied value overrides this
 
-### Outputs
+### Claim Results
+
+The result of executing an operation is defined separately in a claim result document. A claim may have multiple results. For example if the operation was cancelled and then retried the claim will have two results: one representing the cancelled operation and another the retried operation.
 
-The claim `result` provides a list of output names generated by the operation. A tool may choose to request the contents of any outputs to persist them but is not required to do so.
+```json
+{
+  "claimRef": {
+    "installation": "technosophos.helloworld",
+    "revision": "01CP6XM0KVB9V1BQDZ9NK8VP29"
```

Comment:
I don't think this is enough data to ensure that a runtime can resolve a single claim. For instance if two stateless actions are run with different parameters, this would result in two distinct claims but `installation` and `revision` would be the same between each of them.

I'm not sure what the best solution is here; the only ways to ensure correctness as far as I can tell are to embed the entire claim object, or to make claims immutable and reference by content digest. Content digests would also come with the benefit of keeping potentially sensitive data out of results and being small relative to full claim objects.

Reply:
Ah shoot, we completely missed that there isn't anything to reference for `modifies: false` when talking this over last week... Good catch.